### ISSUE TITLE
[New Rule] Non-capital environment variables

### ISSUE THREAD

**Issue by MartinThoma** [MartinThoma/flake8-simplify#19]
## Explanation

It is pretty common to have CONSTANT_STYLE environment variables.

## Example

```python
# Bad
os.environ["foo"]
os.environ.get("bar")

# Good
os.environ["FOO"]
os.environ.get("BAR")
```

**Comment by MartinThoma** [comment]
```
        Expr(
            value=Subscript(
                value=Attribute(
                    value=Name(id='os', ctx=Load()),
                    attr='environ',
                    ctx=Load(),
                ),
                slice=Index(
                    value=Constant(value='foo', kind=None),
                ),
                ctx=Load(),
            ),
        ),
```

**Comment by MartinThoma** [comment]
```
        Expr(
            value=Call(
                func=Attribute(
                    value=Attribute(
                        value=Name(id='os', ctx=Load()),
                        attr='environ',
                        ctx=Load(),
                    ),
                    attr='get',
                    ctx=Load(),
                ),
                args=[Constant(value='bar', kind=None)],
                keywords=[],
            ),
        ),
```

**Comment by MartinThoma** [comment]
Included in v0.10.0
